apps/ai-image-tagging/lambda/package.json

```diff
@@ -7,15 +7,16 @@
   "scripts": {
     "lint": "eslint .",
     "test": "jest",
-    "test:ci": "jest",
     "serve": "node serve.js",
     "deploy": "sls deploy --stage $STAGE",
-    "deploy:test": "npm run deploy"
+    "deploy:test": "npm run deploy",
+    "test:ci": "npm run test:ci:build-frontend && jest",
+    "test:ci:build-frontend": "cd ../frontend && npm run build && cd ../lambda"
```

Comment:
did we end up needing this change after all?

Reply:
I don't think we need to at this point. Nice catch👌